core/src/main/java/org/kiwix/kiwixmobile/core/CoreApp.kt

```diff
@@ -20,23 +20,26 @@ package org.kiwix.kiwixmobile.core
 import android.app.Application
 import android.content.Context
 import android.os.Build
-import android.os.Environment
 import android.os.StrictMode
 import android.os.StrictMode.VmPolicy
-import android.util.Log
 import androidx.appcompat.app.AppCompatDelegate
 import androidx.multidex.MultiDex
 import com.jakewharton.threetenabp.AndroidThreeTen
 import org.kiwix.kiwixmobile.core.data.local.KiwixDatabase
 import org.kiwix.kiwixmobile.core.di.components.CoreComponent
 import org.kiwix.kiwixmobile.core.di.components.DaggerCoreComponent
 import org.kiwix.kiwixmobile.core.downloader.DownloadMonitor
-import java.io.File
-import java.io.IOException
+import org.kiwix.kiwixmobile.core.utils.files.FileLogger
 import javax.inject.Inject
 
 abstract class CoreApp : Application() {
   companion object {
+
+    /* Checks if external storage is available for read and write */
+    @JvmStatic
+    val isExternalStorageWritable: Boolean =
```

Comment:
this similarly needs to use a getter or be a function as right now it gets initialised with this value and accesses simply read the stored value

Reply:
I would prefer if the original code was maintained, that way there will be less files changed in this PR